### DISCUSSION TITLE
Update: extract_struc.py

### DISCUSSION THREAD

**Discussion by Tomoki-YAMASHITA** [Tomoki-YAMASHITA/CrySPY#43] [Announcements]
I updated extract_struc.py in CrySPY Utility (https://github.com/Tomoki-YAMASHITA/CrySPY_utility).

Top k structures (the k most stable structures) can be extracted and output as cif files. 

See the document for details:
https://tomoki-yamashita.github.io/CrySPY_doc/cryspy_utility/extract_struc/

**Comment by yw-fang** [comment]
Thank you for updating it. I didn't use this tool because I wrote another script to extract the structures from opt_POSCARS. Just a very small comment, we may consider the symmetrization tolerance (symprec) as an input parameter (of course, we can put a default value such as 0.01) in this extract_struc.py. Some structures may be very close (they could be the same structure or not), in this case, the tolerance may provide freedom of degree for users to use their intuitions and experience.

**Reply by Tomoki-YAMASHITA** [reply]
Thank you for the comment.

I was going to treat tolerance  as input parameter, but forgot.
I will change it in the next update.
